packages/front-end/components/Features/FeatureModal/index.tsx

```diff
@@ -253,15 +256,14 @@ export default function FeatureModal({
         }
 
         const body = {
-          ...feature,
+          ...featureWithoutHoldout,
           defaultValue: parseDefaultValue(defaultValue, valueType),
-          ...(holdout?.id &&
-            holdout.id !== "none" && {
-              holdout: {
-                id: holdout.id,
-                value: parseDefaultValue(defaultValue, valueType),
-              },
-            }),
+          ...(holdout?.id && {
```

Comment:
Is this still necessary?  Could we just spread `feature` and it will have `{ holdout: {id: "", ...} }` and the back-end will ignore it?